tox.ini

```diff
@@ -0,0 +1,50 @@
+[tox]
+minversion = 3.7.0
+envlist =
+    clear-coverage
+    quality
+    py35
+    py36
+    py37
+    py38
+    compute-coverage
+skip_missing_interpreters = True
+
+[testenv]
+depends =
+    clear-coverage
+# passenv =
+setenv =
+    PYTHONPATH = {toxinidir}/test_cfg
+    DJANGO_SETTINGS_MODULE = settings
+skip_install =
+    coverage: True
+deps =
+    coverage
+commands =
+    coverage run --parallel-mode --source=git_md_page --branch -m django test git_md_page
+
+[testenv:clear-coverage]
+depends =
+commands =
+    coverage erase
+
+[testenv:compute-coverage]
+depends =
+    py{35,36,37,38}
+commands =
+    coverage combine
+    coverage report --include=*/tests/* --fail-under=100 --show-missing
+    coverage report --omit=*/tests/*
```

Comment:
Let's add here `--fail-under=100 --show-missing` as well.

Reply:
Opraveno. Tedy přidáno, ještě budu muset opravit ty faily, co kvůli tomu vznikly... :-(

Reply:
Have you added the tests?

Reply:
Oh, you meant this :D These fails need to be fixed in the second ticket. Now it's okay since the project does not contain any code yet.